src/components/BuySell/components/BuySellButton.tsx

```diff
@@ -4,16 +4,9 @@ import { RoundedButton } from 'components/RoundedButton'
 import useBuySell from 'hooks/useBuySell'
 import useWallet from 'hooks/useWallet'
 import useApproval from 'hooks/useApproval'
-import { Bitcoin2xFlexibleLeverageIndex } from 'constants/productTokens'
 import {
-  daiTokenAddress,
-  usdcTokenAddress,
-  dpiTokenAddress,
-  eth2xfliTokenAddress,
-  btc2xfliTokenAddress,
-  indexTokenAddress,
-  uniswapRouterAddress,
-  sushiswapRouterAddress,
+  ethTokenAddress,
+  zeroExRouterAddress,
```

Comment:
So everything goes through 0x now? Such tidy, much wow

Reply:
Yep! One step closer to migrating to V3...